app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/ProfileFragmentTest.kt

```diff
@@ -0,0 +1,47 @@
+package com.github.sdp.tarjetakuna.ui
+
+import androidx.fragment.app.testing.FragmentScenario
+import androidx.fragment.app.testing.launchFragmentInContainer
+import androidx.lifecycle.Lifecycle
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.sdp.tarjetakuna.R
+import com.github.sdp.tarjetakuna.ui.profile.ProfileFragment
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ProfileFragmentTest {
+
+    private lateinit var scenario: FragmentScenario<ProfileFragment>
+
+    @Before
+    fun setUp() {
+        Intents.init()
+        scenario = launchFragmentInContainer()
+        scenario.moveToState(Lifecycle.State.STARTED)
```

Comment:
Is this line a bit useless?

Reply:
I saw it in a couple tutorials, so I choose to keep it. (I wasn't sur it could have an impact on some config, and since we already had some race condition issues, I prefered to keep it safe.)

Reply:
not really useful as it s the default value of the scenario